apps/barista-design-system/src/components/ba-decision-graph/ba-decision-graph-node/ba-decision-graph-node.html

```diff
@@ -0,0 +1,66 @@
+<div class="ba-uxd-node-wrapper" *ngIf="_node">
+  <div #nodes *ngFor="let node of _decisionGraphSteps">
+    <div class="ba-uxd-node-container">
+      <div class="ba-uxd-node" *ngIf="!node.tasknode">
+        <div
+          class="ba-uxd-node-content"
+          [innerHTML]="getSanitizedNodeText(node.text)"
+        ></div>
+        <div *ngIf="node.path.length">
+          <button
+            class="ba-uxd-edge-button"
+            *ngFor="let edge of node.path.sort()"
+            [ngClass]="
+          {
+            'ba-uxd-yes-edge': edge.text.toLowerCase().includes('yes'),
+            'ba-uxd-no-edge': edge.text.toLowerCase().includes('no')
+          }"
+            [disabled]="edge.selected === false"
+            (click)="setNextNode(edge)"
+          >
+            {{edge.text}}
+          </button>
+        </div>
+      </div>
+    </div>
+    <div class="ba-uxd-tasknode-container" *ngIf="node.tasknode">
```

Comment:
the tasknode container is completly the same as the node container only some classes differ

you could move this out in a template and use it via an ngTemplateOutlet - the classes are applied based on the condition of `node.tasknode`.

Reply:
```
.ba-uxd-tasknode-container {
  width: 100%;
  background-color: $royalblue-600;
  padding: 32px 0;
  margin-top: 32px;
}

.ba-uxd-node-container {
  width: 100%;
}

@media (min-width: $screen-lg) {
  .ba-uxd-node-container {
    width: 76%;
    margin: 0 auto;
  }
}
```